extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -875,6 +872,15 @@ export class AmpA4A extends AMP.BaseElement {
     return true;
   }
 
+  /**
+   * To be overriden by implementing network.
```

Comment:
I worry implementations may accidentally override unlayoutCallback without realizing the consequence.  I suppose its a balancing act between knowing to call super within overridden unlayoutCallback or this.  I slightly prefer super with unlayoutCallback as it makes the lifecycle clear to implementations